website/docs/r/opc_compute_storage_volume_attachment.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "opc"
+page_title: "Oracle: opc_compute_storage_volume_attachment"
+sidebar_current: "docs-opc-resource-storage-volume-attachment"
+description: |-
+  Creates and manages a storage volume attachment in an OPC identity domain.
+---
+
+# opc\_compute\_storage\_volume\_attachment
+
+The `opc_compute_storage_volume_attachment` resource creates and manages a storage volume attachment in an OPC identity domain.
+
+## Example Usage
+
+```hcl
+resource "opc_compute_storage_volume" "default" {
+  name = "storage-volume-1"
+  size = 1
+}
+
+resource "opc_compute_instance" "default" {
+	name = "instance-1"
```

Comment:
Possibly looks like a formatting issue here - might want to re-run the example thru `terraform fmt`.

Reply:
Good call here